src/main/java/org/jabref/gui/SidePaneComponent.java

```diff
@@ -101,8 +101,10 @@ public final Node getHeader() {
         BorderPane graphic = new BorderPane();
         graphic.setCenter(icon.getGraphicNode());
         BorderPane container = new BorderPane();
-        container.setLeft(graphic);
-        container.setCenter(new Label(title));
+//        container.setLeft(graphic);
+        final Label label = new Label(title);
+        label.getStyleClass().add("sidePaneComponentHeader");
```

Comment:
I think it is confusing to use the same style class for two elements. The css selector `sidePaneComponentHeader > .label` should work.